doc/ug/dv_methodology/index.md

```diff
@@ -15,7 +15,7 @@ The following are the key techniques used to perform design verification within
 *  Formal Property Verification (FPV)
 
 For running dynamic simulations, the strategy is to use the [UVM1.2 methodology](https://www.accellera.org/downloads/standards/uvm) on top of a foundation of SystemVerilog based verification to develop constrained-random functional tests.
-Each DUT will include within the repository, a UVM testbench, a testplan, DV plan, a suite of tests, and a method to build, run tests and report the current status.
+Each DUT will include within the repository, a UVM testbench, a [DV plan]({{< relref "doc/ug/dv_methodology/index.md" >}}), overall [DV document]({{< relref "doc/ug/dv_methodology/index.md" >}}), a suite of tests, and a method to build, run tests and report the current status.
```

Comment:
Could you link these to the relevant sub-sections? 
```suggestion
Each DUT will include within the repository, a UVM testbench, a [DV plan]({{< relref "doc/ug/dv_methodology/index.md#dv-plan" >}}), overall [DV document]({{< relref "doc/ug/dv_methodology/index.md#dv-document" >}}), a suite of tests, and a method to build, run tests and report the current status.
```

Reply:
fixed